code/modules/research/designs/medical_designs.dm

```diff
@@ -349,6 +349,39 @@
 	build_path = /obj/item/organ/cyberimp/arm/toolset
 	category = list("Implants", "Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+	
+/datum/design/cyberimp_botanyset
+	name = "Botanical Arm Implant"
+	desc = "A set of botanical tools hidden behind a concealed panel on the user's arm."
+	id = "ci-botanyset"
+	build_type = MECHFAB
+	materials = list (/datum/material/iron = 2500, /datum/material/glass = 1000, /datum/material/silver = 500, /datum/material/plastic = 500)
+	construction_time = 200
+	build_path = /obj/item/organ/cyberimp/arm/toolset
```

Comment:
build_paths are not set

Reply:
lol, you'd think i'd remember arguably the most important part